src/main/java/com/minecolonies/colony/Schematics.java

```diff
@@ -219,7 +291,12 @@ public static int getMaxLevelForHut(String hut)
     @SideOnly(Side.CLIENT)
     public static void setStyles(Map<String, List<String>> hutStyleMap, Map<String, List<String>> decorationStyleMap)
     {
-        Schematics.hutStyleMap = hutStyleMap;
-        Schematics.decorationStyleMap = decorationStyleMap;
+        /*todo this isnt right.  we need to look at the build tool
+        and this methods interaction in order to correct it.
+
+         */
+
+        //Schematics.hutStyleMap = hutStyleMap;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)